src/Http/Http/src/DefaultHttpContext.cs

```diff
@@ -40,13 +46,21 @@ public DefaultHttpContext()
             Features.Set<IHttpResponseBodyFeature>(new StreamResponseBodyFeature(Stream.Null));
         }
 
+        /// <summary>
+        /// Initializes a new instance of the DefaultHttpContext class with options passed in.
```

Comment:
```suggestion
        /// Initializes a new instance of the <see cref="DefaultHttpContext" /> class with the provided features.
```

"Features" are a specific thing in HttpContext, not just options. The Features are what actually implement the HttpContext behavior ;).